src/registry.json

```diff
@@ -348,7 +348,7 @@
     "npm:mina-snap": {
       "id": "npm:mina-snap",
       "metadata": {
-        "name": "Mina"
+        "name": "MinaPortal"
```

Comment:
To be effective, this needs to be changed in the manifest

Reply:
Yes they will publish an update very soon

Reply:
Update published.